apps/studio/data/database-cron-jobs/database-cron-jobs-infinite-query.ts

```diff
@@ -25,36 +25,25 @@ export type CronJob = {
 // [Joshen] Just to call out that I had AI help me with this, so please let me know if this can be optimized
 const getCronJobSql = ({ searchTerm, page }: { searchTerm?: string; page: number }) =>
   `
-WITH latest_runs AS (
-  SELECT 
-    jobid,
-    status,
-    MAX(start_time) AS latest_run
-  FROM cron.job_run_details
-  GROUP BY jobid, status
-), most_recent_runs AS (
-  SELECT 
-    jobid, 
-    status, 
-    latest_run
-  FROM latest_runs lr1
-  WHERE latest_run = (
-    SELECT MAX(latest_run) 
-    FROM latest_runs lr2 
-    WHERE lr2.jobid = lr1.jobid
-  )
-)
-SELECT 
+SELECT
```

Comment:
I'm not 100% sure on this one. It does a nested loop over `job` which makes me a little nervous, but assuming `job` is much smaller than `job_run_details`, it makes much better use of the `job_run_details.runid` index

<img width="1244" height="296" alt="Image" src="https://github.com/user-attachments/assets/d44f5df4-8206-436f-affa-a1f6a5e4a5df" />